test/api.test.js

```diff
@@ -1,8 +1,7 @@
-/* jshint expr: true */
+/* eslint no-unused-expressions: 0 */
```

Comment:
Why not `/* eslint-disable no-unused-expressions */`? It's a bit more readable, especially if one is not very familiar with eslint :=